services/app/pawnguild/registration/templates/confirmation_email.html

```diff
@@ -1,2 +1,19 @@
-<h2> Thanks for registering at PawnGuild! </h2>
-<h4>Please click <a href="{{ link }}">here</a> to confirm your account. Once confirmed, you can create Pawns </h4>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Activate your PawnGuild Account</title>
+</head>
+<body>
+<h1>Thank you for registering on PawnGuild, {{ user.username }}!</h1><br>
```

Comment:
I'm not entirely sure about `{{ user.username }}` as I didn't test it, it does seem like Django provides the user model though. It would also be nice to add some additional fields like time of request and ip address for added security.

Reply:
Works! I thought we'd need `{{ request.user.username }}` but I guess user is accessible directly